app/presenters/two_factor_options_presenter.rb

```diff
@@ -106,11 +111,13 @@ def skip_label
   private
 
   def piv_cac_required?
-    @piv_cac_required
+    @piv_cac_required &&
+      !mfa_policy.piv_cac_mfa_enabled?
   end
 
   def phishing_resistant_only?
-    @phishing_resistant_required && !mfa_policy.phishing_resistant_mfa_enabled?
```

Comment:
Interesting that we already had very similar logic! 🤔  Seems like we were at least partly aware of the desired behavior.

Reply:
yea but we didn't use it or leverage it much lol.